tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -249,3 +327,44 @@ def test_failover_and_relocate(
             )
 
         # TODO: Add data integrity checks
+
+        # Get lastGroupSyncTime after relocate
+        for drpc_data in TimeoutSampler(60, 3, drpc_obj.get):
```

Comment:
The wait time should be at least slightly higher than the sync interval in case of relocate before checking the lastGroupSyncTime.

Reply:
Increased the wait time